source/javascripts/meetups.js

```diff
@@ -45,7 +45,9 @@
 
   handler.buildMap(mapOptions, function() {
     if(navigator.geolocation) {
-      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap);
+      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap, function(){
```

Comment:
I accidentally worked on this as well (sorry) on sandraor#9. This kinda explains the approach I went for, I also split the two actions into different functions.